src/main/java/uk/gov/hmcts/divorce/systemupdate/schedule/migration/task/RemoveAccessCode.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.divorce.systemupdate.schedule.migration.task;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.api.CaseDetails;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.State;
+import uk.gov.hmcts.divorce.divorcecase.task.CaseTask;
+
+@Component
+public class RemoveAccessCode implements CaseTask {
```

Comment:
Would it make more sense renaming it to `RemoveAccessCodeTask` to have consistent naming for tasks?